code/API_definitions/qod-api.yaml

```diff
@@ -329,16 +332,32 @@ components:
       type: object
       required:
         - sessionId
-        - event
+        - eventReport
       properties:
         sessionId:
           $ref: "#/components/schemas/SessionId"
-        event:
+        eventReport:
           $ref: "#/components/schemas/SessionEvent"
     SessionEvent:
-      type: string
-      enum:
-        - SESSION_TERMINATED
+      type: object
+      required:
+        - event
+      properties:
+        event:
+          type: string
+          enum:
+            - QOS_STATUS_CHANGED
+        qosStatus:
+          type: object
+          properties:
+            status:
+              $ref: "#/components/schemas/QosStatus"
```

Comment:
We may need to distinguish the values in the response and this asynchronous event. For example, `requested` may make sense for the 20x response, but not for the notification event.